tools/buildTools/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required( VERSION 2.6 )
 
-
+INSTALL(FILES "robocomp.pth" DESTINATION ~/.local/lib/python3.6/site-packages)
```

Comment:
Ok, we need to make this to not depend on the python version, but I think I have seen something for this.

Reply:
python3 -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])'

Reply:
Yeah, I didn't know how to find the path of site-packages... that's why left the PR incomplete